wiki/Ranking_Criteria/osu!catch/en.md

```diff
@@ -1,5 +1,7 @@
 # osu!catch ranking criteria
 
+**Note: This document is an extension of the [general ranking criteria](https://osu.ppy.sh/wiki/en/Ranking_Criteria).**
```

Comment:
```suggestion
*Note: This document is an extension of the [general ranking criteria](/wiki/Ranking_Criteria).*
```

Reply:
Just pushed a commit with all the things fixed, but thanks for pointing out anyway 👍